app-interact/app-interact-q&a.md

```diff
@@ -42,7 +42,9 @@
 
 ### 8. Care este diferența principală între FIFO și socket UNIX?
 
-   FIFO-urile au un singur canal de comunicare (unidirecțional), pe când socket-urile UNIX prezintă două buffer-e (comunicare bidirecțională), unul pentru send și unul pentru receive.
+   Principala diferență este modul în care sunt utilizate și transmise datele între procese. FIFO se bazează pe sistemul de fișiere, în timp ce socket-ul UNIX se bazează pe o conexiune de rețea între procese.
```

Comment:
Same thing with the previous comment: "Nice to know", can become confusing if the reader is not experimented.
Also, I wouldn't say that "socket-ul UNIX se bazează pe o conexiune de rețea între procese" because that's not entirely true, as sockets (UNIX and Berkeley) use fds to communicate, and that's not reflected in the answer properly. I don't think that this change is necessary.